apps/server/src/api/ws/engine-log-streamer.ts

```diff
@@ -5,8 +5,10 @@ import EventEmitter = NodeJS.EventEmitter;
 export class EngineLogStreamer {
   private cleanup: Function;
   private readonly firstConnectionQuery = {
-    limit: 10,
     start: 0,
+    limit: 10,
```

Comment:
We are showing more than 10 logs in the floating window, why are we limiting logs to latest 10 in the pop out window @fcastill ?

Reply:
Noticed that this limitation is only for the first time when we open either of the logs window